### PR TITLE
Use custom credentials when sending a message

### PR DESCRIPTION
**Pull request by bugant**
I found it useful to customize each request with different api public and private keys, since we are using the library in a multi user setting.

If you think this could be useful I can extend this to the rest of the APIs as well.

### CODE REVIEW

File: lib/mailjex/utils/comms.ex [hunk 1]
```diff
@@ -32,9 +33,9 @@ defmodule Mailjex.Utils.Comms do
     end
   end
 
-  def request(:put, path, body) do
+  def request(:put, path, body, public_key, private_key) do
```
Comment (overture8) [inline lib/mailjex/utils/comms.ex:36]: The PUT version of `request` need some default values so we don't break the current api.

```suggestion
  def request(:put, path, body, public_key \\ nil, private_key \\ nil) do
```

**Review by overture8 — COMMENTED** [review]
Thanks a lot for this. See my comments 👍

File: lib/mailjex/utils/comms.ex (hunk 1 above)
Comment (bugant) [reply lib/mailjex/utils/comms.ex:36]: Ah checked it now, the default values are defined here https://github.com/PiggyPot/mailjex/pull/5/files/c6b109416ce88b7096fc67b95b14d7a4d95856ec#diff-990bfd45dfb16afa2fb76b46d9b25fbaR15 and applies to all definitions of request/5.

**Comment by overture8** [comment]
Sorry for the delay. Merging now.

**Pull request merged**